doc/chromebooks/setting_gbb_flags.md

```diff
@@ -0,0 +1,200 @@
+
+the circut is located on the motherboard
+![screw](./assets/wp-screw/lenovo10e.jpg)
+described in [Hardware Maintanance Manual](https://download.lenovo.com/consumer/mobiles_pub/lenovo_10e_chromebook_tablet_hmm_v2.pdf)
+</details>
+
+_Tip.**you** can **look** for hardware maintanace manual for ur device **urself**, these are sometimes mentioned there_
+
+_Note. if you find one yourself PR is welcome_
+
+## Disconnecting battery
+
+_Warning. opening your chromebook will likely void your warranty and may result in your device being bricked even if you think you know what you are doing_
+
+_Note. this might not work on all chromebooks, for example [ideapad duet](https://github.com/hexdump0815/imagebuilder/issues/53#issuecomment-2197343028)_
+
+1. this option like the one above requires you to open up your device
+2. disconnect the battery from the mainboard and connect
+3. connect the device to a charger 
+_Note. there are minimal voltage requirements for device to turn on, you device doesn't turn on, it's likely your power source is too weak_
+4. turn on your device
+
+as long as the battery is disconnected the write protection should be dissabled on most devices
+
+## Using suzyqable (recommended, modern chromebooks)
+
+in order to perform this you will need suzyqcable to, buy or make one
+
+- https://chromium.googlesource.com/chromiumos/third_party/hdctools/+/HEAD/docs/ccd.md#making-your-own-suzyq
+- https://www.youtube.com/watch?v=WGsyXlgSxFk
+
+enter tty and check if CCD (Closed Case Debugging) is opened
+```
+gsctool -a -I
+```
+
+if it is opened you can skip opening it
+
+but if it isn't run
+```
+gsctool -a -o
+```
+_Note. it will require you to press PP button (a fancy word for power button) a few times, if you screen turns off after pressing that button just press it twice in a row quicklu_
+
+after last press the device will shutdonw and you will have to enable developer mode again
+
+now running
+```
+gsctool -a -I
+```
+should show that ccd is open
+
+now plugin suzyqcable to you chromebook
+
+_Note. it will on work plugged into one of chromebooks ports and in one orientation_
+
+now check if it works
+
+```
+ls /dev/ttyU*
+```
+output
+```
+ttyUSB0 ttyUSB1 ttyUSB2
+```
+_Note. this may not appear on some chromebooks with non chromeos, you can connect the other end diffrent linux pc_
+ 
+_Note. you can connect the device to itself if it has 2 ports_
+
+you can disable hardware writeprotection on any linux/chromeos (as long as device gets detected) with
+
+```
+echo "wp false" > /dev/ttyUSB0
+echo "wp false atboot" > /dev/ttyUSB0
+```
+
+_Note. on debian devices that detect the ttyUSB0 you can use ```minicom /dev/ttyUSB0``` (```sudo apt install minicom```) to see whats going on behind the sceen and type the samec commands, if it complains about modem being missing just ```ln /dev/ttyUSB0 /dev/modem```_
+
+after doing it enable it back just to be safe
+
+```
+echo "wp true" > /dev/ttyUSB0
+echo "wp true atboot" > /dev/ttyUSB0
+```
+
```

Comment:
maybe a note about enabling testmode (see my arm chromebook repo) might be good as well - i think it is also good to keep the chromebook open in some cases (i do not remember exactly what it was good for right now)?